cmd/security-secretstore-setup/res/configuration.toml

```diff
@@ -38,6 +38,7 @@ TokenProviderAdminTokenPath = "/run/edgex/secrets/tokenprovider/secrets-token.js
 PasswordProvider = ""
 PasswordProviderArgs = [ ]
 RevokeRootTokens = true
+ConsulSecretsAdminTokenPath = "/tmp/edgex/secrets/edgex-consul/admin/token.json"
```

Comment:
Don't code the whole path into the executable.  Use existing environment variables.

Reply:
don't get it, what existing env?

Reply:
Sorry, it wasn't an env var.  It was OutputDir, here https://github.com/edgexfoundry/edgex-go/blob/12a188d7528ee379d18cf3e1ac715e90dfef7258/cmd/security-secretstore-setup/res-file-token-provider/configuration.toml#L11

Reply:
ok

Reply:
but that is in fileprovider, nothing to with that utility here

Reply:
resolved